L1Trigger/L1THGCal/src/be_algorithms/HGCalMulticlusteringImpl.cc

```diff
@@ -66,7 +68,9 @@ void HGCalMulticlusteringImpl::clusterize( const edm::PtrVector<l1t::HGCalCluste
     /* making the collection of multiclusters */
     multiclusters.resize(0, multiclustersTmp.size());
```

Comment:
This not correct since only clusters above threshold are filled.
Remove the `resize`  and replace the `set` with
```cpp
multiclusters.push_back( 0, multiclustersTmp.at(i));
```